docs/extensions/my-first-extension/_posts/1970-01-07-WorkingWithData.md

```diff
@@ -6,11 +6,26 @@ section: My first extension
 ---
 
 # Working with Data
-<hr />
 
-Lets fetch data from the Shoutem Cloud storage to the app. First, remove the assets folder, we don't need it anymore. Create a  `reducer.js` file in the `app` folder. This file will contain a `reducer` defining the initial app state and how the state changes.
+Let's fetch data from the Shoutem Cloud storage to the extension. First, remove the `app/assets` folder, we don't need it anymore. Also remove the `getRestaurants()` function from `List.js`.
 
-Our [@shoutem/redux-io](https://github.com/shoutem/redux-io) package has `reducers` and `actions` that communicate with the Shoutem CMS. The `storage` reducer retrieves data (e.g. restaurants) into a dictionary, while `collection` stores data ID's in an array to persist its order.
+```JavaScript{2-4}
```

Comment:
Users regularly leave this function in their code, which causes errors because the function requires a file from the `assets` folder which doesn't exist. I know this because 90% of the builds that fail with a "restaurants" extension fail because of it.